### PR TITLE
docs: clarification in Theme Inheritance usage

### PR DESCRIPTION
**Pull request by anisabboud**
Make it clearer that "extend: '@vuepress/theme-default'" should be in .vuepress/theme/index.js and not .vuepress/config.js.

**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: packages/docs/docs/theme/inheritance.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ For now theme inheritance doesn't support high-order inheritance, that means, a
 
 ## Usage
 
-Suppose you want to create a theme inherited from the default theme, you just need to configure the [extend](./option-api.md#extend) option in your theme configuration:
+Suppose you want to create a theme inherited from the default theme, you just need to configure the [extend](./option-api.md#extend) option in your theme configuration (`.vuepress/theme/index.js`):
```
Comment (ulivz) [inline packages/docs/docs/theme/inheritance.md]: Actually you can modify the `theme configuration` to `theme entry` and  link it to [Glossary > themeEntry](https://v1.vuepress.vuejs.org/miscellaneous/glossary.html#themeentry).

**Comment by f3ltron** [comment]
just up here because the docs need a big update on it with real world example 👍

**Comment by f3ltron** [comment]
i assigned you @sobolevn as you added a label on it if you ok with that

**Comment by sobolevn** [comment]
@f3ltron sure!

**Review by ulivz — APPROVED** [review]

**Comment by ulivz** [comment]
Could you address the conflicts?

**Pull request closed without merging**